torch/nn/utils/parametrize.py

```diff
@@ -282,6 +283,15 @@ def _inject_new_class(module: Module) -> None:
     """
     cls = module.__class__
 
+    def default_deepcopy(self, memo=None):
+        # Just emulate a standard deepcopy procedure when __deepcopy__ doesn't exist in the current class.
+        memo = {} if memo is None else memo
+        replica = self.__new__(self.__class__)
```

Comment:
You should first check if self is not in the memo.

Reply:
Yes, now deepcopy is created only when `id(self) not in memo`